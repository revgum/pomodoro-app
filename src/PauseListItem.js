// @flow
import * as React from "react";
import "./PauseListItem.css";
import * as Utils from "./utils";

type Props = {
  seconds: number,
  description?: string
};

export default class PauseListItem extends React.Component<Props> {
  render() {
    return (
      <li>
        <span>{Utils.formattedTime(this.props.seconds)} : </span>
        <span>{this.props.description}</span>
      </li>
    );
  }
}
