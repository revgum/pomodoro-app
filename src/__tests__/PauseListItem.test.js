import React from "react";
import PauseListItem from "../PauseListItem";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("<PauseListItem />", () => {
  it("passes a smoke test", () => {
    const wrapper = shallow(<PauseListItem />);
  });

  it("captures a snapshot", () => {
    const tree = renderer.create(<PauseListItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
