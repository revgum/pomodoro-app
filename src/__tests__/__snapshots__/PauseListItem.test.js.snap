// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PauseListItem /> captures a snapshot 1`] = `
<li>
  <span>
    NaN:NaN
     : 
  </span>
  <span />
</li>
`;
